holoviews/plotting/mpl/mplwidgets.js

```diff
@@ -19,7 +19,11 @@ var MPLMethods = {
     } else {
       this.update_cache();
     }
-    this.update(0);
+    if (this.dynamic | !this.cached) {
+      this.update(0)
```

Comment:
So does ``update(0)`` refer to the initial key that includes defaults?

Reply:
In the dynamic case that should be the case yes, in the embedded case below it's not, hence adding this branch.